src/vec8bit.h

```diff
@@ -57,6 +57,129 @@ extern Obj FuncASS_VEC8BIT (
     Obj                 pos,
     Obj                 elm );
 
+
+/****************************************************************************
+**
+*F  ZeroVec8Bit( <list> ) . . . make a new 8 bit vector
+**
+*/
+extern Obj ZeroVec8Bit( UInt q, UInt len, UInt mut);
+
+/****************************************************************************
+**
+**  Low-level access, needed for meataxe64 package  
+**
+*/
+
+extern Obj GetFieldInfo8Bit( UInt q);
+
+
+
+/****************************************************************************
+**
+*F  LEN_VEC8BIT( <vec> ) . . . . .. . . . . . . .length of an 8 bit GF vector
```

Comment:
Insert missing spaces between `..` and also in `.length` ? Or if that's too annoying, just leave out those lines ;-).